src/bricks/wallet/types.ts

```diff
@@ -84,10 +84,10 @@ export interface IWalletBrickCustomization {
 
 interface IWalletBrickCheckoutCustomization {
   /**
-   * Optional. Wallet Brick offers two optional customizable theme variables that receive hexadecimal values:
+   * Optional. Wallet BrickBrick offers two optional customizable theme variables that receive hexadecimal values:
```

Comment:
Foi um `Brick` a mais haha
```suggestion
   * Optional. Wallet Brick offers two optional customizable theme variables that receive hexadecimal values:
```